node_exporter.go

```diff
@@ -114,15 +118,40 @@ func loadCollectors(list string) (map[string]collector.Collector, error) {
 	return collectors, nil
 }
 
+// versionInfoTmpl contains the template used by printVersion.
+var versionInfoTmpl = `
+node_exporter, version {{.version}} (branch: {{.branch}}, revision: {{.revision}})
+  build user:       {{.buildUser}}
+  build date:       {{.buildDate}}
+  go version:       {{.goVersion}}
+`
+
+// printVersion rint version information.
+func printVersion() {
+	t := template.Must(template.New("version").Parse(versionInfoTmpl))
+
+	var buf bytes.Buffer
+	if err := t.ExecuteTemplate(&buf, "version", version.Map); err != nil {
+		panic(err)
+	}
+	fmt.Fprintln(os.Stdout, strings.TrimSpace(buf.String()))
```

Comment:
I'm acutally against this kind of printing. These are the only lines not being in the log format. Let's use our standard log format to print all information.

And yes I'm aware that this comes from the prometheus server.